### ISSUE TITLE
TerraFirmaCraft HQM 

### ISSUE THREAD

**Issue by skushnarev** [lorddusk/HQM#182]
Hello!
HQM does not start automatically when installed TerraFirmaCraft.
In the configuration files, everything is set up. When I disconnect TFC, HQM works as it should.
Just everything is working correctly when I use HQM command.
When creating a new world, the chat is displayed:

This server doesn't have Hardcore Questing Mode enabled.

Minecraft 1.7.10
Forge 10.13.4.1558
HQM 4.4.4
TFC 0.79.28.908

**Comment by skushnarev** [comment]
@Dream-Master I know. But I need to be enabled automatically.

```
# If set to true, new worlds will automatically activate Questing mode
B:"Auto-start questing mode"=true
```

**Comment by Dream-Master** [comment]
Do it one time and its enabled forever.

**Comment by skushnarev** [comment]
@Dream-Master If you are not an experienced player will launch, he will not know how to turn on.
It should work as it is set in the configuration file.

**Comment by Dream-Master** [comment]
@skushnarev 
Did you know that before I ask there are no option to turn off the hardcore mode when enabled once?
The command are not listed (I did not see it) if you Type /hqm help
(/hqm hardcore disable)
With some mods you play some try and error game.

**Comment by skushnarev** [comment]
@Dream-Master I'm not interested. I need to work properly autorun.

**Comment by lorddusk** [comment]
If an unexperienced player can't read he will have trouble yes.
It is shown in big letters how to enable questing mode...

Also. TFC is known to not play nice with certain mods.

**Comment by way2muchnoise** [comment]
Closed due to no longer supporting 1.7.10, please check if this problem persists in the latest 1.9.4 version
